src/ert/config/queue_config.py

```diff
@@ -83,6 +83,20 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
             ):
                 submit_sleep = float(values[0])
 
+        def _is_simulator(fm_name: str):
```

Comment:
Why do we restrict the tagging to selected forward model names? That is not mentioned in the documentation for PROJECT_CODE.

Reply:
As far as I understood the idea is to tag job submissions to each of the drivers with a code that can be used to identify what resources are being used. For tracking purposes. 

I will add this information to the queue documentation.